.travis.yml

```diff
@@ -26,7 +26,7 @@ matrix:
 deploy:
```

Comment:
The folks over at adafruit have merged my proposal so you can start using their script again (I'll remove mine). 
So please replace `platisd` with `adafruit` at L16.